Framework/Geometry/inc/MantidGeometry/Rendering/ShapeInfo.h

```diff
@@ -65,28 +66,44 @@ class MANTID_GEOMETRY_DLL ShapeInfo {
     double radius;
     double height;
   };
+  struct HollowCylinderGeometry {
+    const Kernel::V3D &centreOfBottomBase;
+    const Kernel::V3D &axis;
+    double innerRadius;
+    double outerRadius;
+    double height;
+  };
 
 private:
   std::vector<Kernel::V3D> m_points;
-  double m_radius; ///< Radius for the sphere, cone and cylinder
-  double m_height; ///< height for cone and cylinder;
+  double m_radius;      ///< Radius for the sphere, cone and cylinder
+  double m_height;      ///< height for cone, cylinder and hollow cylinder;
+  double m_innerRadius; ///< Inner radius for hollow cylinder
+  double m_outerRadius; ///< Outer radius for hollow cylinder
```

Comment:
Maybe we could reuse `m_radius` for the outer radius to not to bloat the class with extra members?

Reply:
That's a sensible thing to do. I'll do that.